my-thai-star/index.asciidoc

```diff
@@ -0,0 +1,28 @@
+= My Thai Star
+====
+My-Thai-Star is the Open Source reference application for devonfw. This tutorial will walk you through the process of running My-Thai-Star on docker.
+
+## Prerequisites
+* User should have angular and Java development experience.
+
+## Learning goals
+* In this tutorial you will learn to run My-Thai-Star application on docker.
+
+More information about My-Thai-Star can be found at https://devonfw.com/website/pages/docs/master-my-thai-star.asciidoc.html
+====
+
+First you need to clone the My Thai Star repository.
+[step]
+--
+cloneRepository("my-thai-star","https://github.com/devonfw/my-thai-star.git")
+--
+
+Now you need to switch to the folder you just created.
+
+`cd my-thai-star`{{execute T1}}
+
+After that, you can build and run the application using docker-compose. 
+[step]
+--
+dockerCompose("my-thai-star", { "startupTime": 6000, "port": 8082, "path": "" })

```

Comment:
change port to 8081 and lower startup time. 100 minutes is way too much. It should work with the default value of 10 minutes, so you can remove the parameter startupTime

Reply:
docker-compose did not work in 8081 port and 6mins startup time. Check the previous runs.

Reply:
8081 should be the right port. Maybe the run with port 8081 failed because the startup time was too low. Can you increase it a bit (not too much)? Maybe 20 minutes (1200 seconds)